core/api/src/services/ledger/index.ts

```diff
@@ -414,7 +459,9 @@ export const LedgerService = (): ILedgerService => {
     fns: {
       updateMetadataByHash,
       getTransactionById,
+      getTransactionForWalletById,
       getTransactionsByHash,
+      getTransactionsForWalletByPaymentHash,
```

Comment:
Small optional nit to be consistent with other methods here
```suggestion
      getTransactionByWalletIdAndId,
      getTransactionsByHash,
      getTransactionsByWalletIdAndPaymentHash,
```